docs/guides/dataloaders.md

```diff
@@ -160,7 +160,7 @@ async def load_users(keys) -> List[User]:
 class MyGraphQL(GraphQL):
     async def get_context(self, request: Union[Request, WebSocket]) -> Any:
```

Comment:
Good catch! While you're at it, could you also add the missing typing imports (Any, List, Union, ...)? This would make these examples fully copy-pasteable :)

Reply:
Sure, added them now

Reply:
Thanks!